### ISSUE TITLE
Data files for reproduction of association rule learning

### ISSUE THREAD

**Issue by mgermy** [aleda145/product_allocation_aframe#10]
Hi @aleda145! 

First, I want to say congrats for the thesis! It's very well written and the brings nice insights regarding product allocation. I am trying to reproduce a part of the methodology you used in your thesis (association rule learning). However, I couldn't figure out how the following files are structured: aframe_channels.csv & num_channels_per_sku_median.csv

Is aframe_channels.csv a csv with the max capacity of an aframe on each row? 

Could you maybe share these two files, so I can understand better how it's structured (if it's not a problem and not confidencial, of course)? Or maybe just a similar file with random numbers, so you don't have to share the real numbers. 

Best regards, 
Michel

**Comment by aleda145** [comment]
Hi! 
Thank you!

Sorry that the code is a bit messy, I should have spent more time making it easier to understand the steps I took, but I was ready to drop the project after finishing the thesis!

`aframe_channels.csv` was a csv file that I did not check into git actually and it was a confidential file on how the case company handled the A-frame in their system. I have also wiped the computer it was on, so I can't give you a definitive answer on this. But I'm pretty sure it had a structure like:

```
channel_id
0
1
2
3
```
and so on for all channels. So what the allocation scripts would do is just append the sku_id to the a specific channel, so the `aframe_allocation.csv`

Would look like:

```
channel_id, sku_id
0, 123
1, 123
2, 4040
```

Hope it helps! You are welcome to ask more questions!
